webpack.config.js

```diff
@@ -28,8 +28,10 @@ const hubspotConfig = ({ portal, autoupload } = {}) => {
         {
           test: /\.s[ac]ss$/i,
           use: [
+            "style-loader",
```

Comment:
Do we need both `MiniCSSExtractPlugin` and `style-loader`? Since we don't really have a devmode in this webpack config, I was under the impression the extract plugin takes its place.